minio/minio.go

```diff
@@ -0,0 +1,191 @@
+package minio
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+)
+
+// Storage interface that is implemented by storage providers
+type Storage struct {
+	minio *minio.Client
+	cfg   Config
+	ctx   context.Context
+}
+
+// New creates a new storage
+func New(config ...Config) *Storage {
```

Comment:
The `Reset` flag is not being check/used in `New()`

Reply:
The Reset flag added to New flag